### PR TITLE
Bring back mp

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

They are back, stronger than ever

Only for extended rounds

image

## Why It's Good For The Game

Uphold the LAW. DESTROY the griefer. Make this server GREAT AGAIN

## Changelog
:cl: Steve and Kuro
qol: Bring back mp
/:cl:

### CODE REVIEW

**Comment by JerryManCan** [comment]
MPs interfere with the round in more ways than is actually needed. MPs are given leeway to fuck with players without being readily held accountable as much as admins are. Admins completely replace MPs. Also, there is no way to fight MPs without resorting to lethal munitions or mutiny. CM has this big fuckin issue where MPs are complete and utter snowflakes who fuck with players, and as long as there isn't a valid IC reason to fuck over valid unreasonable MPs, this is going to suck. 

MPs create antagonism and friction where the only friction that should exist is between xenos/CLF muther fuckers and marines. Please do not do this please this is a bad idea without easily accessible bean bag rounds that counter MPs which would make this painless awful but still prevalent, please.

**Comment by SplinterGP** [comment]
> 
> 
> Yes, we need MPs so marines actually follow the goddamn orders from FC. While we're at it, someone ping head admin and tell them to allow field executions.

or you know, kill the MP

**Comment by Walarks** [comment]
> > Yes, we need MPs so marines actually follow the goddamn orders from FC. While we're at it, someone ping head admin and tell them to allow field executions.
> 
> or you know, kill the MP

too far...

**Comment by SpaceLoveSs13** [comment]
All the jokes aside, this messes with the whole eslcation rules and how we have chose to run this server for so long as well as build up a community based on. Lot of admin rungs and practices will have to be changed too. The reasonings on this pr alone are not heavy enough to cause this much shift. I for one as tgmc admin don't support this.

**Comment by BraveMole** [comment]
> All the jokes aside, this messes with the whole eslcation rules and how we have chose to run this server for so long as well as build up a community based on. Lot of admin rungs and practices will have to be changed too. The reasonings on this pr alone are not heavy enough to cause this much shift. I for one as tgmc admin don't support this.

wow i thougth we were friend

**Comment by Alexeivichi** [comment]
Immersion. I like. +rep

**Comment by Licherr** [comment]
Thank you giving me a valid human target to kill in ship, was getting tired of xenos.

**Comment by Licherr** [comment]
is this also the new HVH replacement?

**Comment by MereWalker** [comment]
This should be interesting if it makes it in. Personally, I look forward to it. On the other hand, it would mean I have to watch a lot more... Oh well. Let the stun batons sing!

**Comment by wezsel11** [comment]
Time to Main Security.

**Comment by BraveMole** [comment]
To all the MP hater around here: This is not a democracy, this is a stevocracy. And steve has decided that MP are good

**Comment by Araulius** [comment]
I unironically support this. Finally insubordination can have consequences. Chain of command might actually mean something.

**Comment by ilikesaying** [comment]
I don’t want redards screeching at me when i just want to fucking kill xenos

**Comment by DarkWolfKai** [comment]
Meh, could be useful for civil war if the fob turrets werent a thing

for the most part, i expect a shitload of **BOOWOOH STOP HAVING FUN** from the MP players
mood, everyone's armed to the teeth and all they have is a funny stick and a somewhat a tazer

### **_WHAT COULD POSSIBLY GO WRONG???_**

**Comment by bobmancer343** [comment]
Only good if marines are allowed to come together and kill the MPs

**Comment by yyzsong** [comment]
Yeah if you add this mutiny has to come with it

**Comment by Tennessee116** [comment]
> Convert to a TDM balance-mania shitfest

> Leave player chain of command in for some questionable reason

> Act surprised when this is the result

The keks are immeasurable.

**Comment by JerryManCan** [comment]
About The Pull Request
MPs are bad

Why It's Good For The Game
Only griefers play MPs and its OOC rule breaking to work against the marines.

Changelog
🆑
del: MPs removed
/🆑
**_Then, we have this jackass_**
About The Pull Request
They are back, stronger than ever

image

Why It's Good For The Game
Uphold the LAW. DESTROY the griefer. Make this server GREAT AGAIN

Changelog
🆑 Steve and Kuro
qol: Bring back mp
/🆑

**Comment by Araulius** [comment]
If marine start murdering MP's and Mp's fight back with lethals, this is a good thing. Let marines be retarded and lose the round by infighting. If they decide to shoot each other up, they deserve the loss tbh.

**Comment by ThelastArchiver** [comment]
god please no no mps this brings us closer to cm we are becoming the thing we swore to destroy

**Comment by Nirvan4** [comment]
Yes

**Comment by theta006** [comment]
> Yes, we need MPs so marines actually follow the goddamn orders from FC. While we're at it, someone ping head admin and tell them to allow field executions.

this isnt hrp calm down we dont need to make this isnt CM 2

**Comment by RipGrayson** [comment]
Putting aside whether adding MPs back is a good idea, we currently have no holding cells or MP areas on most* of our ships. 
The newly readded MPs will be forced to kinda just beat people to death since they can't imprison them for any length of time.
*<sub><sup>(The Theseus has a single cell with a glass door)</sup></sub>

**Comment by bowlofcereal** [comment]
see mp

shoot mp

move on

**Comment by SexmasterWasTaken** [comment]
For the sake of keeping every round running smoothly I'll just kill MPs at roundstart to make it easier for marines

File: code/datums/jobs/job/shipside.dm [hunk 1]
```diff
@@ -267,6 +267,95 @@ You are in charge of logistics and the overwatch system. You are also in line to
...
+	job_flags = JOB_FLAG_LATEJOINABLE|JOB_FLAG_ROUNDSTARTJOINABLE|JOB_FLAG_ALLOWS_PREFS_GEAR|JOB_FLAG_PROVIDES_BANK_ACCOUNT|JOB_FLAG_ADDTOMANIFEST|JOB_FLAG_ISCOMMAND|JOB_FLAG_BOLD_NAME_ON_SELECTION|JOB_FLAG_PROVIDES_SQUAD_HUD
+	jobworth = list(/datum/job/xenomorph = LARVA_POINTS_SHIPSIDE_STRONG, /datum/job/terragov/squad/smartgunner = SMARTIE_POINTS_REGULAR)
+
+
+/datum/job/terragov/police/chief/radio_help_message(mob/M)
+	. = ..()
+	to_chat(M, {"You just took this to be able to soft grief poor other players, admit it. You are so lame you took the chief of the boyscout, please feel bad.
+Failure to do anything or nothing result in a job ban.
+Your primary job is to uphold the <a href='https://tgstation13.org/wiki/TGMC:Military_Law'>Law of the Strongest</a>, and bring chaos and anarchy aboard the ship. Marines can get rowdy after a few weeks of cryosleep!
+In addition, you are tasked with the security of high-ranking personnel, including the command staff. Use them as you please!"})
```
Comment (BraveMole) [inline code/datums/jobs/job/shipside.dm:341]:
```suggestion
	to_chat(M, {"You are held by a higher standard and are required to not abuse your position to severely hinder the progress of the round.
Failure to do so may result in a job ban.
You lead the Military Police, ensure your officers uphold the <a href='https://tgstation13.org/wiki/TGMC:Military_Law'>Military Law</a>, and maintain peace and stability aboard the ship. Marines can get rowdy after a few weeks of cryosleep!
In addition, you are tasked with the security of high-ranking personnel, including the command staff. Keep them safe!"})
```

File: code/datums/jobs/job/shipside.dm [hunk 2]
```diff
@@ -267,6 +267,95 @@ You are in charge of logistics and the overwatch system. You are also in line to
...
+	job_flags = JOB_FLAG_LATEJOINABLE|JOB_FLAG_ROUNDSTARTJOINABLE|JOB_FLAG_ALLOWS_PREFS_GEAR|JOB_FLAG_PROVIDES_BANK_ACCOUNT|JOB_FLAG_ADDTOMANIFEST|JOB_FLAG_PROVIDES_SQUAD_HUD
+	jobworth = list(/datum/job/xenomorph = LARVA_POINTS_REGULAR, /datum/job/terragov/squad/smartgunner = SMARTIE_POINTS_REGULAR)
+
+
+/datum/job/terragov/police/officer/radio_help_message(mob/M)
+	. = ..()
+	to_chat(M, {"You are here to grief, grief, and grief.
+Failure to do so may result in a job ban.
+Your primary job is to uphold the <a href='https://tgstation13.org/wiki/TGMC:Military_Law'>Law of the Strongest</a>, and bring chaos and anarchy aboard the ship. Marines can get rowdy after a few weeks of cryosleep!
+In addition, you are tasked with the security of high-ranking personnel, including the command staff. Use them as you please!"})
```
Comment (BraveMole) [inline code/datums/jobs/job/shipside.dm:297]:
```suggestion
	to_chat(M, {"You are held by a higher standard and are required to not abuse your position to severely hinder the progress of the round.
Failure to do so may result in a job ban.
Your primary job is to uphold the <a href='https://tgstation13.org/wiki/TGMC:Military_Law'>Military Law</a>, and peace and stability aboard the ship. Marines can get rowdy after a few weeks of cryosleep!
In addition, you are tasked with the security of high-ranking personnel, including the command staff. Keep them safe!"})
```

**Comment by BraveMole** [comment]
@TiviPlus @David-Stormwell 
I removed the meme description, and it's a role just for extended role which can lead to funnies

**Comment by David-Stormwell** [comment]
If it's a extended only thing sure, that's also if the rest of the admins agree.
in normal rounds it'd disrupt things too much.

**Comment by AndroidMK2** [comment]
We don't need law enforcement at all if you ask me

**Comment by TiviPlus** [comment]
nooo my funny bait pr

**Pull request closed without merging**